readme_pages/getting_cugraph.md

```diff
@@ -0,0 +1,58 @@
+
+# Getting cuGraph Packages
+
+There are 4 ways to get cuGraph packages:
+1. [Quick start with Docker Repo](#docker)
+2. [Conda Installation](#conda)
+3. [Pip Installation](#pip)
+4. [Build from Source](../SOURCEBUILD.md)
```

Comment:
please fix the link as I get a 404

Reply:
SOURCEBUILD should be a single "." since it is in the same directory - good catch

Reply:
fixed